packages/features/auth/lib/getServerSession.ts

```diff
@@ -46,9 +46,11 @@ export async function getServerSession(options: {
     return cachedSession;
   }
 
-  const user = await prisma.user.findUnique({
+  const user = await prisma.user.findFirst({
     where: {
       email: token.email.toLowerCase(),
+      username: token.username,
+      organizationId: token.org?.id,
```

Comment:
Providing the `orgId` and `username` from the current token to better select the user in the database, as those fields along with `email` will make it unique.